mcdreforged/info_reactor/impl/server_reactor.py

```diff
@@ -27,3 +40,4 @@ def react(self, info):
 			if handler.test_server_stopping(info):  # notes that it might happen more than once in the server lifecycle
 				self.mcdr_server.logger.debug('Server stopping detected', option=DebugOption.REACTOR)
 				self.mcdr_server.rcon_manager.disconnect()
+				self.mcdr_server.server_info.clear()
```

Comment:
I'll suggest to reset the info on server started

Reply:
In my mind, a down server has no these information, so I thought this should be reset on server stop

Reply:
Then you should reset the info on server **stopped**, not on server **stopping**